playbooks/openstack/configuration.md

```diff
@@ -171,17 +171,103 @@ another external DNS server.
 
 ## Kuryr Networking Configuration
 
-If you opt to use Kuryr for networking, make sure that you review all
-the Kuryr options. The following parameters should be uncommented in
-`inventory/group_vars/all.yml`:
+Kuryr is an SDN that uses OpenStack Neutron. This prevents the double overlay
+overhead one would get when running OpenShift on OpenStack using the default
+OpenShift SDN.
+
+
+### OpenStack Requirements
+
+Kuryr has a few additional requirements on the underlying OpenStack deployment:
+
+* The Trunk Ports extension must be enabled:
+  * https://docs.openstack.org/neutron/pike/admin/config-trunking.html
+* Neutron must use the Open vSwitch firewall driver:
+  * https://docs.openstack.org/neutron/pike/admin/config-ovsfwdriver.html
```

Comment:
Maybe same thing here with 'neutron-openvswitch-agent'? I know this may seem obvious, but it always takes me a minute or two to find the right service to restart, and then I'm never immediately sure that I've restarted the right thing.